### DISCUSSION TITLE
Gui version?

### DISCUSSION THREAD

**Discussion by yazze** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#719] [Q&A]
So I used to use the old UAD version which was straight-forward, click on the file and it launches GUI, period. Since it hasn't been updated, and it regularly fucks the package lists, I wanted to try a newer version/fork. But this one doesn't have any gui, it's just a file, and there's no way to launch it through term, which let me hold you for a second, I WILL NOT BOTHER with a pos non-gui or unnecessary steps to do a task as simple as launching as I have very little time and the old UAD worked.

So any idea where I can find a gui version that works?

**Comment by AnonymousWP** [comment]
Can you post logs? Launching works for most people, but only a handful of people have the `wgpu` issues.
